easybuild/easyconfigs/d/ddt/ddt-7.0.5.eb

```diff
@@ -10,64 +10,74 @@ toolchain = {'name': 'dummy', 'version': 'dummy'}
-#       optional : LD_LIBRARY_PATH
-# set    --> will set the variable for the script, nothing more
-# setenv --> will export the variable to the user environment
-# ============================================================================
-set     url                        "http://www.allinea.com"
-set     url2                       "http://user.cscs.ch"
-set     modulename                 "ddt"
-# ============================================================================
-if  { $hostname=="esch" ||$hostname=="kesc" ||$hostname=="leon" ||$hostname=="monc" } {
-          set     os      "RHAT6"
-          set     version "7.0"
-}
-  
-if  { $hostname=="ela1" ||$hostname=="ela2" ||$hostname=="ela3" } {
-          set     os      "SLES11"
-          set     version "7.0"
+if { [ file exist /etc/os-release ] } {
+
+    set fsize [file size /etc/os-release ]
+    set fp [open /etc/os-release r]
+    set OS_INFO [read $fp $fsize]
+
+    foreach i $OS_INFO {
+        set key [ lindex [ split $i '=' ]  0 ]
+        set value [ lindex [ split $i '=' ]  1 ]
+        # puts stderr "i=$i : $key === $value"
+
+        if { $key == "ID" } {
+            set OS_ID [ string trim $value '"' ]
+        }
+
+        if { $key == "VERSION_ID" } {
+            set VERSION_ID [ string trim $value '"' ]
+            set VERSION_ID_MAJOR [ lindex [ split $VERSION_ID '.' ]  0 ]
+        }
+
+    }
 }
-  
-if  { $hostname=="dain" ||$hostname=="dom1" ||$hostname=="nid0" } {
-          set     os      "SLES12"
-          set     version "7.0"
+
+#puts stderr "OS_ID=${OS_ID}"
+#puts stderr "VERSION_ID=$VERSION_ID"
+#puts stderr "VERSION_ID_MAJOR=$VERSION_ID_MAJOR"
+
+if { ${OS_ID}=="sles" } {
+    set ost "SLES"
+    if { ${VERSION_ID_MAJOR}=="12" } {
+        set osv "12"
+    } else {
+        set osv "11"
+    }
 }
 
-if  { $hostname=="tave" } {
-          set     os      "SLES12"
-          set     version "7.0"
+if { ${OS_ID}=="rhel" } {
+    set ost "RHAT"
+    if { ${VERSION_ID_MAJOR}=="7" } {
+        set osv "7"
+    } else {
+        set osv "6"
+    }
 }
-  
-if { "$os"=="" } {
-        puts stderr "L60: host=$host"
-        set     os      "SLES12"
-        set     version "7.0"
+
+if { ${OS_ID}=="generic" } {
+    set ost "RHAT"
+    set osv "6"
 }
 
-        set     packageroot             "/apps/common/UES/${os}/${modulename}/${version}"
+set     url          "http://www.allinea.com"
+set     url2         "http://user.cscs.ch"
+set     modulename   "ddt"
+set     version      "7.0.5"
+set     packageroot  "/apps/common/UES/${ost}${osv}/${modulename}/${version}"
 
 # ============================================================================
-# module-info mode ==> load, remove, display, help, whatis, switch, switch1, switch2, or switch3.
-# load
-# ============================================================================
 #if { [ module-info mode load ] } {
```

Comment:
You do not need the `if { [ module-info mode load ] } {`. It is even commented out.
Why not delete it?